src/features/apiDelegation/offered/ChooseApiPage/ChooseApiPage.tsx

```diff
@@ -128,12 +124,12 @@ export const ChooseApiPage = () => {
         key={index}
```

Comment:
Question: Why remove the actionBarWrapper div on the delegatabe apis but not on the chosen ones?